frontend/src/error/ErrorHandler.tsx

```diff
@@ -1,13 +1,13 @@
 import React from "react";
 import { useLocation } from "react-router-dom";
 import { get } from "lodash";
-import { ErrorPage } from "./ErrorPage";
+import { BackendErrorPage } from "./ErrorPages";
 
 const ErrorHandler = ({ children }: any) => {
   const location = useLocation();
```

Comment:
Hmm... curious if we can harness typescript to avoid spelling issues with "errorStatusCode" by typing the location state.

Does the following work by any chance?
```tsx
type LocationState = {
    errorStatusCode: number;
}

const { state } = useLocation<LocationState>()
const { errorStatusCode } = state;
```